### DISCUSSION TITLE
No Audio/No Mode Change

### DISCUSSION THREAD

**Discussion by RadioRD** [AlexandreRouma/SDRPlusPlus#376] [Q&A]
I'm running SDR++ v1.0.3 on Win10 with a SDRPlay RSPdx.  SDR++ connects to the receiver, signals appear on the display, the receiver can be tuned and the antenna can be selected.  However the speaker symbol in the upper left of the display has a slash through it, the volume slider can not be moved and no audio can be heard.  Clicking on the speaker symbol does not remove the slash.  Under Radio, Audio is set to Audio .  The speaker symbol at that location does not have a slash and the volume slider slides.  I suspect I may be missing something obvious but so far have not been able to find my error.  All advise appreciated.

![SDRPP](https://user-images.githubusercontent.com/91234435/134446457-07a85890-0abe-47e2-80b8-f168454a6e2d.jpg)

**Comment by AlexandreRouma** [comment]
I don't see any VFO on the FFT, you likely disabled the radio module by unchecking the checkbox on its menu. But you didn't include that menu in the screenshot so I can't say more.

**Reply by RadioRD** [reply]
Thanks Alexandre.  Clearly I still have a lot to learn.  When you mention the lack of VFO on the FFT I'm not sure where I should be looking for that.  Regarding the radio module question, I see SDRplay Source listed on the Module Manager menu.  But clicking on what I take to be the checkbox to the right stops the receiver operation and selects airspy_source.  To review, SDR++ finds the SDRplay receiver, tunes the receiver, allows antenna ports to be changed and shows signals on the waterfall display.  But I am unable to hear audio or change mode
![SDRPP-02](https://user-images.githubusercontent.com/91234435/134607485-4bbce1f4-5b86-460f-95c8-f0f85f85fd68.jpg)
![SDRPP-03](https://user-images.githubusercontent.com/91234435/134607501-806aabca-8426-40a1-b46d-921c8adbc00c.jpg)
s
![SDRPP-04](https://user-images.githubusercontent.com/91234435/134607505-fa504500-18f9-4f6d-b385-98846e6e8dc2.jpg)
.

**Reply by the-gobleo-tinkerer** [reply]
There is a checkbox at the right end of the "▼ Radio" tab. In your screenshot it's unchecked, meaning the radio (demodulator) is disabled and not giving any audio output. Normally you want this box to be checked.

What I think you did is you removed (by clicking on the - minus sign) the SDRPlay Source module from the module manager.

**Comment by RadioRD** [comment]
On Fri, Sep 24, 2021 at 2:08 AM Shuyuan Liu ***@***.***>
wrote:

> There is a checkbox at the right end of the "▼ Radio" tab. In your
> screenshot it's unchecked, meaning the radio (demodulator) is disabled and
> not giving any audio output. Normally you want this box to be checked.
>
> What I think you did is you removed (by clicking on the - minus sign) the
> SDRPlay Source module from the module manager.
>

That's it!  Thanks so much for pointing out the obvious to a new user.

Regards,

RD

=========================
Richard Dillman
Inverness Ridge, CA
=========================
